dhall-format/Main.hs

```diff
@@ -0,0 +1,80 @@
+{-# LANGUAGE DataKinds          #-}
+{-# LANGUAGE DeriveGeneric      #-}
+{-# LANGUAGE ExplicitNamespaces #-}
+{-# LANGUAGE OverloadedStrings  #-}
+{-# LANGUAGE TypeOperators      #-}
```

Comment:
In non-Haskell stuff, I like scripts to have doc comments at the top saying:
- what it is
- why you might want to use it
- an example of how to use it

I like this in Haskell stuff too, but I always forget how to write the correct Haddock markup. ;)